<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPsicologo</class>
 <widget class="QMainWindow" name="MenuPsicologo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Psicologo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="creaUsuario">
       <attribute name="title">
        <string>Crear</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Nombre</string>
          </property>
          <property name="buddy">
           <cstring>nombre</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="apellidos"/>
        </item>
        <item row="10" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Crear</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="telefono"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="folioLabel">
          <property name="text">
           <string>Folio</string>
          </property>
          <property name="buddy">
           <cstring>folio</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="mailLabel">
          <property name="text">
           <string>eMail</string>
          </property>
          <property name="buddy">
           <cstring>email</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="apLabel">
          <property name="text">
           <string>Apellidos</string>
          </property>
          <property name="buddy">
           <cstring>apellidos</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="email"/>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="nombre"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="folio"/>
        </item>
        <item row="6" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="telLabel">
          <property name="text">
           <string>Telefono</string>
          </property>
          <property name="buddy">
           <cstring>telefono</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="direccionLabel">
          <property name="text">
           <string>Direccion</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="direccion"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modificaUsuario">
       <attribute name="title">
        <string>Modificar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QLineEdit" name="mFolioLineE"/>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="mBuscarPB">
             <property name="maximumSize">
              <size>
               <width>85</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="mFolio">
             <property name="text">
              <string>Folio:</string>
             </property>
             <property name="buddy">
              <cstring>mFolioLineE</cstring>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="mFolioLabel">
             <property name="text">
              <string>Folio</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="radioFolio">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="folio_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="nameLabel_2">
             <property name="text">
              <string>Nombre</string>
             </property>
             <property name="buddy">
              <cstring>nombre</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="radioNombre">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="nombre_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="apLabel_2">
             <property name="text">
              <string>Apellidos</string>
             </property>
             <property name="buddy">
              <cstring>apellidos</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="radioApellidos">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="telLabel_2">
             <property name="text">
              <string>Telefono</string>
             </property>
             <property name="buddy">
              <cstring>telefono</cstring>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="radioPhone">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="telefono_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="mailLabel_2">
             <property name="text">
              <string>eMail</string>
             </property>
             <property name="buddy">
              <cstring>email</cstring>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QCheckBox" name="radioEMail">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="email_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="14" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="apellidos_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="direccion_2">
             <property name="text">
              <string>Direccion</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="eliminaUsuario">
       <attribute name="title">
        <string>Eliminar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="eFolio"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="eFolioPB">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="eFolioLabel">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Folio:</string>
          </property>
          <property name="buddy">
           <cstring>eFolio</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="buscaUsuario">
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QPushButton" name="mostrar">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="mostrarTable"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>folio</tabstop>
  <tabstop>nombre</tabstop>
  <tabstop>apellidos</tabstop>
  <tabstop>telefono</tabstop>
  <tabstop>email</tabstop>
  <tabstop>direccion</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>mFolioLineE</tabstop>
  <tabstop>mBuscarPB</tabstop>
  <tabstop>radioFolio</tabstop>
  <tabstop>folio_2</tabstop>
  <tabstop>radioNombre</tabstop>
  <tabstop>nombre_2</tabstop>
  <tabstop>radioApellidos</tabstop>
  <tabstop>apellidos_2</tabstop>
  <tabstop>radioPhone</tabstop>
  <tabstop>telefono_2</tabstop>
  <tabstop>radioEMail</tabstop>
  <tabstop>email_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>eFolio</tabstop>
  <tabstop>eFolioPB</tabstop>
  <tabstop>mostrar</tabstop>
  <tabstop>mostrarTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
